vioquine/assignment3/WordSearchTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
```

Comment:
Avoid importing whole packages. It is always better to import only what you use, otherwise you might sometimes run into problems with double imports etc.